projects/kit/components/input-files/test/input-files.directive[#4215].spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
```

Comment:
I don't think this test makes sense if it's a platform specific behavior. I believe we can drop it.

Reply:
fixed